src/test/components/__snapshots__/ProfileCallTreeView.test.js.snap

```diff
@@ -4705,7 +4705,7 @@ exports[`calltree/ProfileCallTreeView renders call tree with some search strings
             style={
               Object {
                 "height": "96px",
-                "width": "3000px",
+                "width": 3000,
```

Comment:
Do you think it could be possible to have a tree exercizing the formula in a test?

Reply:
Easy enough, will do.